k8s/security_context.go

```diff
@@ -8,6 +8,17 @@ import (
 
 var oneTwentyFour = semver.MustParse("1.24")
 
+func defaultPodSecurityContext() *corev1.PodSecurityContext {
+	// change ownership of the mounted volume to the first non-root user uid=1000
+	runAsUser := int64(1000)
```

Comment:
This should be set only for non-OpenShift.

Reply:
In your opinion, the best way to achieve this?